lib/editor/components/ExceptionValidationErrorsList.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import React from 'react'
+
+import { getComponentMessages } from '../../common/util/config'
+import type { EditorValidationIssue } from '../util/validation'
+
+const VALIDATION_ERROR_LIMIT = 3
+
+const ExceptionValidationErrorsList = (props: {validationErrors: Array<EditorValidationIssue>}) => {
+  const messages = getComponentMessages('ExceptionValidationErrorsList')
+  const { validationErrors } = props
```

Comment:
Move this to the function declaration.

Reply:
Good call, updated as of https://github.com/ibi-group/datatools-ui/pull/965/commits/cb8061a21d0716ace8efaaada0b845e7c594a703